### PR TITLE
Feature/new toolchain

### PR DESCRIPTION
**Pull request by magwas**

### CODE REVIEW

File: metamodel.zenta [hunk 1]
```diff
@@ -1533,46 +1526,15 @@ count(zenta:neighbours($p1,$p2,'contains,1')[@xsi:type='Entity']) = 1</documenta
...
-              </child>
-         </element><element xmlns:html="http://www.w3.org/TR/xhtml1/transitional" xmlns:fn="http://www.w3.org/2005/xpath-functions" xsi:type="zenta:BasicObject" name="Called by at least 1 Service" id="75c88dc9-f8b4-4315-8bf4-7989930c2f55" ancestor="bb3cdf1a-0bc1-4772-8913-d0567ef7ab1f" parent="ac65e2e9-896e-4c2c-a4c9-3ac7cb0c49f9">
-            <documentation>design|
-count(zenta:neighbours($p1,$p2,'operates on,2')[@xsi:type='Service']) &gt; 0</documentation>
-         </element></folder>
+            <element xsi:type="zenta:BasicObject" name="Entity" id="dc030e1c-7c6c-4479-8030-c001aa814e9d" ancestor="41b4d079-dbd5-4df2-91db-6a596661ab0d"/>
+            <element xsi:type="zenta:BasicObject" name="Only CRUD or existing query operations" id="0fbc4f57-24ed-4896-895e-71a1b8179290" ancestor="bb3cdf1a-0bc1-4772-8913-d0567ef7ab1f">
+              <documentation>design|
+count(zenta:neighbours($p1,$p2,'operates on,2')[@xsi:type='Service']) > 0</documentation>
```
Comment (szirbucz) [inline metamodel.zenta]: This is not the correct expression for this contract.

File: metamodel.zenta [hunk 2]
```diff
@@ -1496,28 +1481,36 @@ count(zenta:neighbours($p1,$p2,'uses,1')[@xsi:type='HTTP method']) = 1</document
             <property key="font"/>
           </element>
         </folder>
-        <folder name="06. Repository" id="0af3528f-c853-4ded-b956-9e5197174d9e">
+        <folder name="07. Repository" id="0af3528f-c853-4ded-b956-9e5197174d9e">
```
Comment (szirbucz) [inline metamodel.zenta]: Repository should be joined to Entity type

**Review by szirbucz — CHANGES_REQUESTED** [review]

File: metamodel.zenta (hunk 1 above)
Comment (magwas) [reply metamodel.zenta]: Do you mean that the name of the Constraint should be changed to be more expressive, or that the xpath expression should be changed?

File: metamodel.zenta (hunk 2 above)
Comment (magwas) [reply metamodel.zenta]: ACK

File: metamodel.zenta (hunk 1 above)
Comment (szirbucz) [reply metamodel.zenta]: The XPath should be changed according to the name

**Review by szirbucz — APPROVED** [review]

**Comment by szirbucz** [comment]
I can't squash and merge this

**Pull request merged**